### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by colinksmith**
I updated the readme.md for testing purposes

### CODE REVIEW

**Review by GUIMess — CHANGES_REQUESTED** [review]
+ should be = and -

**Comment by colinksmith** [comment]
> * should be = and -

What about it?

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@ Repo for collaboration on a social app that will be used to recommend tv and str
 
 Start your server with "npm run dev", terminal should tell you what localhost you need!
 
+testing
```
Comment (GUIMess) [inline README.md:6]: are we sure it's testing on this line?

**Review by GUIMess — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (colinksmith) [reply README.md:6]: yes that is testing on that line

**Review by GUIMess — APPROVED** [review]
lgtm

**Pull request merged**